core/sequencer/sequencer_api.proto

```diff
@@ -75,8 +75,18 @@ message DefineRevisionsRequest {
 
 // DefineRevisionResponse contains information about freshly defined revisions.
 message DefineRevisionsResponse {
-  // outsanding_revisions a list of all the defined revisions which are not yet applied.
-  repeated int64 outstanding_revisions = 1;
+  // map_revision is the current map revision that was fully applied. Any
+  // unapplied revisions are greater than map_revision.
+  int64 map_revision = 2;
```

Comment:
`highest_applied_revision`?

Reply:
How about `applied_to`?

Reply:
Renamed to `highest_applied`. I think the "revision" word is redundant.